curvefs/src/client/common/config.cpp

```diff
@@ -104,6 +104,27 @@ void InitExcutorOption(Configuration *conf, ExcutorOpt *opts) {
                               &opts->enableRenameParallel);
 }
 
+void InitExcutorInternalOption(Configuration *conf, ExcutorOpt *opts) {
+    conf->GetValueFatalIfFail("excutorOpt.maxInternalRetry", &opts->maxRetry);
+    conf->GetValueFatalIfFail("excutorOpt.retryIntervalUS",
```

Comment:
If other parameters are reused, this function can also be reused directly.

`void InitExcutorOption(bool internal, ...)`

Reply:
fixed